pydatalab/pydatalab/routes/v0_1/collections.py

```diff
@@ -359,6 +359,8 @@ def add_items_to_collection(collection_id):
                     "relation": None,
                     "type": "collections",
                     "immutable_id": ObjectId(collection["_id"]),
+                    "item_id": None,
+                    "refcode": None,
```

Comment:
Perhaps we should use the actual relationships model here? I can't use the suggestion but it would look something like

```
TypedRelationship(description="Is a member of", type="collections", immutable_id=ObjectId(collection["_id"]))
```

then all of the `None` fields will be set automatically. Perhaps we even want to use a specialisation of this for collection relationships in future, that sets the type and ID for you